addons/test_website/static/tests/tours/image_link.js

```diff
@@ -39,7 +39,7 @@ wTourUtils.registerWebsitePreviewTour('test_image_link', {
     ...selectImageSteps,
     {
         content: "check popover content has site URL",
-        trigger: ".o_edit_menu_popover a.o_we_url_link[href='http://odoo.com/']:contains(http://odoo.com/)",
+        trigger: "iframe .o_edit_menu_popover a.o_we_url_link[href='http://odoo.com/']:contains(http://odoo.com/)",
```

Comment:
Yes, there is the same problem as for the manipulator's, with its CSS conflicting with the theme's one.

The problem today is that bootstrap popovers are not meant to be used "on top" of iframes.
In our most standard use case: desktop edition in LTR language, it works because the iframe is placed with the same offset and dimensions as the container's ownerDocument*.
In other cases: edition in RTL languages, mobile mode, it does not because the iframe has different offset and dimensions than the container's ownerDocument.

I guess that Bootstrap uses the container's ownerDocument to compute placements, and does not take into account whether or not the target is located inside an iframe (therefore, skipping the iframe's offset and dimensions in the placements computations).

The options Boostrap provides us with are not helpful :
- the offset will place the tooltip correctly, but the arrow is not updated, still pointing to the element where it would be placed without the iframe's offset
![image](https://user-images.githubusercontent.com/58561718/196391409-12f29bfc-8b59-412d-9cf6-88ce2e47be93.png)

- also, placing the popover outside of the iframe leads to the same problem as the manipulators before https://github.com/odoo/odoo/commit/872bb20b3ac08cf82613e15e6634a2e7593ccf7a : it overflows when scrolling 
![image](https://user-images.githubusercontent.com/58561718/196391475-d7ff6e2d-0842-45c5-8536-808e3c43f330.png)

Maybe we can have a discussion on the popover's specs, and see what can be done or not based on the Bootstrap's popover ?

*: the container being the HTMLElement that we give as an option to initialize the popover: for now, either the target's ownerDocument body, or the parent of its frameElement. In any case, the container's ownerDocument will be the global document.

Reply:
Ok, let's keep in the iframe for now, we'll reconsider to do better in master. Any way to move it inside of the #oe_manipulators element (which is supposed to contain every "overlay" related stuff)? And maybe add some more css rules to make it look like the current one (which is out of the iframe)? :+1:

Reply:
Changed according to the last comment, added css rules and put it in the `#oe_manipulators` div. It is ready for review